docker/run_tests.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -ex
+## Pull in /etc/os-release so we can see what we're running on
+. /etc/os-release
+
+## Default Vars
+CLONE_DIRECTORY='/github_actions_build'
+CTEST_TESTING_DIRECTORY='build'
+GTEST_TESTING_DIRECTORY='build/testing'
+RUN_CTEST=true
+RUN_GTEST=true
+
+usage() {
+  set +x
+  echo ""
+  echo "Usage: ${0}"
+  echo ""
+  echo "Optional Arguments:"
+  echo "  -c : Run only the ctest suite"
+  echo "  -g : Run only the gtest suite"
+  echo "  -D <clone/directory>: The absolute path to the directory where Performous"
+  echo "     is cloned to from GitHub. Defaults to ${CLONE_DIRECTORY}"
+  echo "  -t <test/directory>: The relative path to the testing directory for google tests"
+  echo "     under the clone of the Performous repo. Defaults to ${GTEST_TESTING_DIRECTORY}"
+  echo "  -T <test/directory>: The relative path to the testing directory for 'make test'"
+  echo "     under the clone of the Performous repo. Defaults to ${CTEST_TESTING_DIRECTORY}"
+  echo "  -h : Show this help message"
+  exit 1
+}
+
+## Set up getopts
+while getopts "cgD:t:T:h" OPTION; do
+  case ${OPTION} in
+    "c")
+      RUN_GTEST=false;;
+    "g")
+      RUN_CTEST=false;;
+    "D")
+      CLONE_DIRECTORY=${OPTARG};;
+    "t")
+      GTEST_TESTING_DIRECTORY=${OPTARG};;
+    "T")
+      CTEST_TESTING_DIRECTORY=${OPTARG};;
```

Comment:
We might wanna switch up these options since it's kind of a bad practice to have capitalized letters do a different thing.

What might even be better is to have one "testing" directory which gets used by both ctest/gtest so we end up with one option: `TESTING_DIRECTORY`

Reply:
Ok, the options are switched. I agree with the testing for both being in the same place, but not sure I'd be able to do it properly on my own. I think we should make an issue, and enlist the help of @twollgam who originally made the testing stuff :-)